CHANGELOG.md

```diff
@@ -6,65 +6,141 @@ SPDX-License-Identifier: MIT OR Apache-2.0 WITH LLVM-exception
+## [1.0.0](https://github.com/kokkos/kokkos-fft/tree/1.0.0) (2025-12-18)
 
-**Implemented enhancements:**
+**Documentation:**
+
+- Rename kokkos-fft to Kokkos-FFT[\#388](https://github.com/kokkos/kokkos-fft/pull/388)
+- Align prerequisites with Kokkos 5.0[\#387](https://github.com/kokkos/kokkos-fft/pull/387)
+- Add docstring to get_map_axes[\#340](https://github.com/kokkos/kokkos-fft/pull/340)
+- Add a link to JOSS paper [\#304](https://github.com/kokkos/kokkos-fft/pull/304)
+
+**CI:**
+
+- Add spack spec and fix install-test in spack CI [\#324](https://github.com/kokkos/kokkos-fft/pull/324)
+- Add spack installation CI for SYCL backend [\#323](https://github.com/kokkos/kokkos-fft/pull/323)
+- Add spack installation CI for HIP backend [\#318](https://github.com/kokkos/kokkos-fft/pull/318)
+- Update rocm Dockerfile to use hipcc 6.2.0 [\#316](https://github.com/kokkos/kokkos-fft/pull/316)
+- Update C++ version to 20 in nightly workflow [\#314](https://github.com/kokkos/kokkos-fft/pull/314)
+- Add spack installation CI for CUDA backend [\#312](https://github.com/kokkos/kokkos-fft/pull/312)
+
+**General Enhancement:**
+
+- Update to kokkos4.6 [\#385](https://github.com/kokkos/kokkos-fft/pull/385)
+- use std::size_t to manipulate extents [\#384](https://github.com/kokkos/kokkos-fft/pull/384)
+- Refactor normalization and unit-tests [\#381](https://github.com/kokkos/kokkos-fft/pull/381)
+- Introduce layout header [\#380](https://github.com/kokkos/kokkos-fft/pull/380)
+- Move get_map_axes into KokkosFFT_Mapping.hpp [\#377](https://github.com/kokkos/kokkos-fft/pull/377)
+- Expose DynPlans [\#376](https://github.com/kokkos/kokkos-fft/pull/376)
+- Add dynamic plan for SYCL backend [\#373](https://github.com/kokkos/kokkos-fft/pull/373)
+- Add dynamic plan for ROCM backend [\#372](https://github.com/kokkos/kokkos-fft/pull/372)
+- Add dynamic plan for HIPFFT backend [\#371](https://github.com/kokkos/kokkos-fft/pull/371)
+- Rephrase the static assertion message regarding data accessibility [\#370](https://github.com/kokkos/kokkos-fft/pull/370)
+- Rename ger_r2c_extent and make it public [\#369](https://github.com/kokkos/kokkos-fft/pull/369)
+- Add dynamic plan for CUDA backend [\#366](https://github.com/kokkos/kokkos-fft/pull/366)
+- Add extents test for dynamic rank case [\#364](https://github.com/kokkos/kokkos-fft/pull/364)
+- Improve iteration pattern in transpose [\#355](https://github.com/kokkos/kokkos-fft/pull/355)
+- Rename iType to IndexType to align with mdspan convention [\#351](https://github.com/kokkos/kokkos-fft/pull/351)
+- Copy if map is identical in transpose helper [\#345](https://github.com/kokkos/kokkos-fft/pull/345)
+- Make transpose helper to work on Views with different layout [\#344](https://github.com/kokkos/kokkos-fft/pull/344)
+- Introduce safe transpose [\#339](https://github.com/kokkos/kokkos-fft/pull/339)
+- Fix the underflow/overflow behavior in total_size function [\#331](https://github.com/kokkos/kokkos-fft/pull/331)
```

Comment:
If this fixes something introduced after 0.4 then i don't think it should be mentioned.

Reply:
OK, I will remove them
#344 is only related to cuFFTMp support for distributed. Currently, it has no impact